src/controllers/signup.js

```diff
@@ -0,0 +1,47 @@
+const Joi = require('joi');
+const {
+  addUser,
+} = require('../database/queries/addData');
+const hashPassword = require('../helpers/hashPass');
+const {
+  signUpSchema,
+} = require('./../helpers/validation-shemas');
+
+exports.get = (req, res) => {
+  res.render('signup', {
+    layout: 'login_signup',
+    title: 'signup ',
+    js: ['collectData', 'signup'],
+    css: ['signup'],
+  });
+};
+
+exports.post = (req, res) => {
+  const user = req.body;
+  const UserInfo = {
+    ...user.firstSection,
+    ...user.secondSection,
+    ...user.thirdSection,
+  };
+  console.log(UserInfo.password);
+  const {
+    error,
+  } = Joi.validate(UserInfo, signUpSchema);
+  if (!error) {
```

Comment:
@Fatmasiam You could add an `else` block to this if-statement and return a 400 (with the validate's error)